src/legacy/core_plugins/kibana_react/public/top_nav_menu/top_nav_menu.test.tsx

```diff
@@ -54,30 +55,51 @@ describe('TopNavMenu', () => {
   ];
 
   it('Should render nothing when no config is provided', () => {
-    const component = shallowWithIntl(<TopNavMenu name="test" uiSettings={setupMock.uiSettings} />);
+    const component = shallowWithIntl(
+      <TopNavMenu
+        name="test"
+        uiSettings={setupMock.uiSettings}
+        savedObjectsClient={startMock.savedObjects.client}
```

Comment:
OK in terms of review, 

@rudolf will savedObjects client be available only in start?
If so, does this mean that TopNavMenu should be rendered only in the start phase?

Reply:
@lizozom Thank you for reviewing again!